### DISCUSSION TITLE
Custom landscapes of Jupiter

### DISCUSSION THREAD

**Discussion by somecoolguy1397** [Stellarium/stellarium#3500] [Show and tell]
I have made custom landscapes using Blender. They show approximate views of the planet Jupiter, from various heights.

![Screenshot_406](https://github.com/Stellarium/stellarium/assets/98181950/5552cb6b-5417-40ba-9cba-e5b22bfabf6d)

[jupiterDeep.zip](https://github.com/Stellarium/stellarium/files/13284587/jupiterDeep.zip)
[jupiterShallow.zip](https://github.com/Stellarium/stellarium/files/13284588/jupiterShallow.zip)

**Comment by gzotti** [comment]
Not bad, but I have a few quibbles.

The description files are no more informative than the minimal description string in landscape.ini. Why not take the chance to explain how you have modelled it (and a bit more than "with Blender". Any scientific foundations, models, papers?), what the vertical darker zones are indicating, or other things about Jupiter of potential interest for the users. Are the location coordinates relevant or random? ("we are located in the South (whatever it is named) zone/belt, ...")
The "deep" landscape is a bit speckled. Should the clouds not be uniform/fog-like?

And please rename so the "Jupiter" is at begin.

**Reply by somecoolguy1397** [reply]
Thanks for the recommendations. I have improved the view, now there is a more clear distinction between the ammonia ice and the troposphere. The location is picked deliberately, as in the big dark zone, now at the horizon, is actually the Great Red Spot. The speckled landscape is a problem with the Blender's Cycles render engine, since it uses ray tracing. The vertical dark zones is a problem that I haven't yet figured out.

[jupiter.zip](https://github.com/Stellarium/stellarium/files/13301617/jupiter.zip)
